opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -670,6 +672,13 @@ typedef Eigen::Array<double,
                     sd_.soMax = fluid_.satOilMax();
                     fluid_.getGasOilHystParams(sd_.krnswdc_go, sd_.pcswmdc_go, cells_);
                     fluid_.getOilWaterHystParams(sd_.krnswdc_ow, sd_.pcswmdc_ow, cells_);
+
+                    sd_.Pb = fluid_.bubblePointPressure(cells_,
```

Comment:
Turns out I was mistaken, I did not notice that this was inside the if-active-gas block.